crates/cli/src/bin/wasm-bindgen-test-runner/node.rs

```diff
@@ -88,7 +98,14 @@ pub fn execute(
     ",
     );
 
-    let js_path = tmpdir.join("run.js");
+    let js_path = if module_format {
+        // fixme: this is a hack to make node understand modules
+        let package_json = tmpdir.join("package.json");
+        fs::write(&package_json, r#"{"type": "module"}"#).unwrap();
+        tmpdir.join("run.mjs")
+    } else {
+        tmpdir.join("run.cjs")
```

Comment:
Please enlighten me: the output we currently produce with the "node" (not the experimental module) target is CommonJS?

Reply:
Yes.

Reply:
Well, another thing to consider in the next breaking change.
Thanks!